packages/teams-js/src/public/runtime.ts

```diff
@@ -223,6 +223,9 @@ interface IRuntimeV4 extends IBaseRuntime {
   readonly isNAAChannelRecommended?: boolean;
   readonly isLegacyTeams?: boolean;
   readonly supports: {
+    readonly app?: {
+      readonly notifySuccessResponse: boolean;
```

Comment:
I don't think there's a reason we can't use a boolean instead of {} here except for convention, but let's use {} because that's what we do for other capabilities/subcapabilities

Reply:
(really, it's because we want to be able to build hierarchies of nested capabilities if needed)

Reply:
Sure, it makes sense!